proto/scene-format.proto

```diff
@@ -0,0 +1,77 @@
+syntax = "proto3";
+
+package com.github.computer_graphics_course;
+
+message Scene {
+	enum MaterialType {
+		SOLID = 1;
+		GLASS = 2;
+		// other materials
+	}
+	message Vector3 {
+		double x = 1;
+		double y = 2;
+		double z = 3;
+	}
+	message Color {
+		double r = 1;
+		double g = 2;
+		double b = 3;
+	}
+	message Transform {
+		int32 id = 1;
+		Vector3 position = 2;
+		Vector3 rotation = 3;
+		Vector3 scale = 4;
+		int32 parent_id = 5;
+	}
+	message Cube {
+		int32 id = 1;
+		Vector3 upper_bounds = 2;
+		Vector3 lower_bounds = 3;
+	}
+	message Sphere {
+		int32 id = 1;
+		double radius = 2;
+	}
+	message Plain {
+		int32 id = 1;
+		Vector3 positon = 2;
+		Vector3 normal = 3;
+	}
+	message MeshedObject {
+		int32 id = 1;
+		string reference = 2;
+	}
+	message SceneObject {
+		int32 id = 1;
+		Transform transform = 2;
+		MaterialType material_type = 3;
```

Comment:
Let's rename it to `material`. By the way, is material defined by its type only?

Reply:
See [related comment by @Brookoko](https://github.com/computer-graphics-course/scene-format/pull/1#discussion_r619599604)

Reply:
I think we can use the same structure as with meshes. This structure makes adding new material very easy.